src/Column/DataColumnRenderer.php

```diff
@@ -198,6 +207,7 @@ public function renderBody(ColumnInterface $column, Cell $cell, DataContext $con
         }
 
         return $cell
+            ->addClass($column->bodyClass)
             ->addAttributes($attributes)
```

Comment:
Change order `addAttributes` and `addClass`, because `addAttributes` override `class` attribute. Also need test for such case.